src/irj_parser/dune

```diff
@@ -0,0 +1,10 @@
+(env
+ (static
+  (ocamlopt_flags
+   (-O3 -ccopt -static))))
```

Comment:
Are we sure we want `-O3` ? That is pretty agressive

Reply:
This stanza is there in a lot of places and only affect static profile builds (only uses for releases IIRC).

Reply:
Just checking, are we leaving this as is? I changed the rest following your remarks (thanks for those!) and it's ready to be pushed

Reply:
Yes you can leave this as is :) Please push!